tests/clusters/test_cluster.py

```diff
@@ -102,12 +103,15 @@
 ]
 
 
-def teardown_function() -> None:
+@pytest.fixture(autouse=True)
+def setup_teardown(clickhouse_db: None) -> Generator[None, None, None]:
```

Comment:
stuff like this really sucks: I have to explicitly depend on the clickhouse_db fixture so that it is setup before this fixture, and torn down after it

this matters a lot for fixtures where some test data is inserted